apis/generate.go

```diff
@@ -13,7 +13,7 @@
 //go:generate go run ../legacy/convert/main.go ../legacy/charts/vela-core-legacy/crds
 
 //go:generate go run ../hack/crd/update.go ../charts/vela-core/crds/standard.oam.dev_podspecworkloads.yaml ../legacy/charts/vela-core-legacy/crds/standard.oam.dev_routes.yaml
-
+//go:generate go run ../test/compatibility-test/convert/main.go ../charts/vela-core/crds ../test/compatibility-test/testdata
```

Comment:
we can move it to makefile and just run for test

Reply:
Good idea, will update